EKF/control.cpp

```diff
@@ -339,13 +337,12 @@ void Ekf::controlExternalVisionFusion()
 
 void Ekf::controlOpticalFlowFusion()
 {
-	// Detect if the vehicle is on the ground and is being excessively tilted, shaken or rotated.
+	// Check if motion is un-suitable for use of optical flow
 	if (!_control_status.flags.in_air) {
-		bool motion_is_excessive = ((_accel_mag_filt > 10.8f)
-					    || (_accel_mag_filt < 8.8f)
-					    || (_ang_rate_mag_filt > 0.5f)
-					    || (_R_to_earth(2, 2) < 0.866f));
-
+		bool motion_is_excessive = ((_accel_mag_filt > 14.7f)
+					    || (_accel_mag_filt < 4.9f)
+					    || (_ang_rate_mag_filt > _params.flow_rate_max)
+					    || (_R_to_earth(2,2) < 0.866f));
```

Comment:
Where are these values from? Are they hardware specific?

Reply:
Can we also have those "magic" numbers as documented defines?